cinema/views.py

```diff
@@ -1,45 +1,113 @@
-from rest_framework.decorators import api_view
+from rest_framework import generics, viewsets, status, mixins
+from django.shortcuts import get_object_or_404
+
 from rest_framework.response import Response
-from rest_framework import status
+from rest_framework.views import APIView
 
-from django.shortcuts import get_object_or_404
+from cinema.models import Movie, Actor, Genre, CinemaHall
+from cinema.serializers import (
+    MovieSerializer,
+    ActorSerializer,
+    GenreSerializer,
+    CinemaHallSerializer,
+)
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+
+class ActorList(
+    generics.ListAPIView,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
```

Comment:
The `ActorList` class unnecessarily combines `ListAPIView` with `ListModelMixin` and `CreateModelMixin`. Since `ListAPIView` already provides the `list` functionality, you can use `CreateAPIView` instead to handle both listing and creation without explicitly defining `get` and `post` methods.